nemo/collections/asr/modules/activations.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import torch.nn as nn
+
+__all__ = ['Swish']
+
+
+class Swish(nn.Module):
```

Comment:
Swish is already defined here - https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/parts/jasper.py#L260

Reply:
Should I move this one there or use this one? Activation looks like common modules?

Reply:
I think use that one, since it already exists (just importing it here should be fine).